var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -132,3 +133,9 @@ class NetlibScalapack(ScalapackBase):
     depends_on("fortran", type="build")  # generated
     # versions before 2.0.0 are not using cmake and requires blacs as
     # a separated package
+
+    def url_for_version(self, version):
+        if version >= Version("2.2.0"):
```

Comment:
Eliminate one version instantiation with:
```suggestion
        if self.satisfies("@2.2.0:"):
```

Reply:
it does not work with self.satisfies. The CI reports an error otherwise.

Reply:
I think the correct syntax is `self.spec.satisfites`?